app/mailers/audit_mailer.rb

```diff
@@ -0,0 +1,19 @@
+class AuditMailer < ApplicationMailer
+  helper :audits, :layout
+
+  def summary(options = {})
+    raise ::Foreman::Exception.new(N_("Must specify a user with email enabled")) unless (user=User.find(options[:user])) && user.mail_enabled?
+    time     = options[:time] ? %(time > "#{options[:time]}") : 'time > yesterday'
+    set_url
+    set_locale_for(user)
+    @query    = options[:query] ? "#{options[:query]} and #{time}" : "#{time}"
+    @limit   = options[:limit] || Setting[:entries_per_page]
+    @audits  = Audit.joins_authorized_as(user, Audit, :view_audit_logs).search_for(@query).limit(@limit)
```

Comment:
no need for joins here, just use authorized_as(user, :view_audit_logs, Audit) or set User.current